ThePensionsRegulator.Frontend/HtmlGeneration/ComponentGenerator.TprSearchFooterLinks.cs

```diff
@@ -0,0 +1,76 @@
+﻿using GovUk.Frontend.AspNetCore;
+using GovUk.Frontend.AspNetCore.Extensions.HtmlGeneration;
+using Microsoft.AspNetCore.Html;
+using Microsoft.AspNetCore.Mvc.Formatters;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.AspNetCore.Mvc.ViewFeatures;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ThePensionsRegulator.Frontend.HtmlGeneration
+{
+    public partial class ComponentGenerator
+    {
+        internal const string DefaultShowMoreQuestionsContent = "Show more questions";
+        public virtual TagBuilder GenerateTprSearchFooterLinks(TprSearchFooterLinks footerLinks)
+        {
+            var outer = new TagBuilder("footer");
+
+            var list = new TagBuilder("ul");
+            list.AddCssClass("govuk-list");
+            list.AddCssClass("tpr-search-results__links");
+
+            outer.InnerHtml.AppendHtml(list);
+
+
+            if (footerLinks.Links.Any())
+            {
+                foreach(var link in footerLinks.Links)
+                {
+                    var li = CreateLink(link.Attributes, link.Content);
+                    list.InnerHtml.AppendHtml(li);
+                }
+            }
+            else
+            {
+                var builder1 = new HtmlContentBuilder();
+                builder1.Append("Show more questions");
```

Comment:
Should this use your default constant?

I know you said a further update is still to come, but this text needs to be localisable to Welsh, as does "Visit automatic enrolment Q&As" below.

At this point in my review it feels like `<tpr-search-footer-links></tpr-search-footer-links>` should just allow HTML content, or links as content...